examples/internal/proto/examplepb/echo_service.swagger.json

```diff
@@ -473,46 +473,13 @@
             "required": true,
             "type": "string"
           },
-          {
-            "name": "lineNum",
-            "in": "query",
-            "required": false,
-            "type": "string",
-            "format": "int64"
-          },
-          {
-            "name": "status.progress",
-            "in": "query",
-            "required": false,
-            "type": "string",
-            "format": "int64"
-          },
-          {
-            "name": "status.note",
-            "in": "query",
-            "required": false,
-            "type": "string"
-          },
```

Comment:
I think these were removed incorrectly, no? There's nothing in this RPC definition that means the status query parameters shouldn't show up, I think? Here's the definition: https://github.com/grpc-ecosystem/grpc-gateway/blob/442bd3ab9497e4f9b3bbcbf4e53854300a68dd5a/examples/internal/proto/examplepb/echo_service.proto#L68. I think it's removing all oneofs instead of just the `lineNum` oneof.